Update uninstaller documentation. (#13627)
netdata/netdata: packaging/installer/UNINSTALL.md

> issues with your Netdata Agent installation, consider our [**reinstall Netdata**
> doc](/packaging/installer/REINSTALL.md) instead of removing the Netdata Agent entirely.

The recommended method to uninstall Netdata on a system is to use our kickstart installer script with the `--uninstall` option like so:

```sh
wget -O /tmp/netdata-kickstart.sh https://my-netdata.io/kickstart.sh && sh /tmp/netdata-kickstart.sh --uninstall
```

Or (if you have curl but not wget):

```sh
curl https://my-netdata.io/kickstart.sh > /tmp/netdata-kickstart.sh && sh /tmp/netdata-kickstart.sha --uninstall
```

> This will work in most cases without the user needing to do anything more other than accept removal of configuration

Reviewer: ```suggestion
This will work in most cases without you needing to do anything more other than accepting removal of configuration
```

and data files. You can confirm whether this approach will work for you by adding `--dry-run` to the list of
options. If that produces a line with a message like `Would attempt to uninstall existing install`, then this
method will work on your system.

If you used a non-standard installation prefix, you may need to specify that prefix using the `--old-install-prefix`
option when uninstalling this way.

## Unofficial installs

If you used a third-party package to install Netdata, then the above method will usually not work, and you will
need to use whatever mechanism you used to originally install Netdata to uninstall it.

## Uninstalling manually

Most official installs of Netdata include an uninstaller script that can be manually invoked instead of using the
kickstart script (internally, the kickstart script also uses this uninstaller script, it just handles the process
outlined below for you).

> This uninstaller script is self-contained other than requiring a `.environment` file that was generated ruing

Reviewer 2: @Ferroin Did you mean to say "was generated **during** installation" here?

    3.1 **Interactive mode (Default)**

>     The default mode in the uninstaller script is **interactive**. This means that the script provides the user

Reviewer: ```suggestion
    The default mode in the uninstaller script is **interactive**. This means that the script provides you
```
